lib/modules/datasource/crate/index.ts

```diff
@@ -319,31 +319,17 @@ export class CrateDatasource extends Datasource {
           { clonePath, registryFetchUrl },
           `Cloning private cargo registry`,
         );
-
-        const git = Git({
-          ...simpleGitConfig(),
-          maxConcurrentProcesses: 1,
-        }).env(getChildEnv());
-        const clonePromise = git.clone(registryFetchUrl, clonePath, {
-          '--depth': 1,
-        });
+        const clonePromise = CrateDatasource.clone(
+          registryFetchUrl,
+          clonePath,
+          packageName,
+          cacheKeyForError,
+        );
 
         memCache.set(
           cacheKey,
           clonePromise.then(() => clonePath).catch(() => null),
```

Comment:
```suggestion
          clonePromise,
```

see my changed clone method